README.md

```diff
@@ -29,6 +29,106 @@ Now the `kale` command should be available. Try:
 
     kale help
 
+### Example run
+
+Here is an unedited run through of provisioning commands to prepare a
+Solr collection as a target for running the Data Crawler.
+
+After downloading `kale` from
+[releases](https://github.com/IBM-Watson/kale/releases) I opened a
+terminal window. The `$` is my command prompt. I typed each command
+shown on the lines starting with `$`. All other text is output from
```

Comment:
Do you think we could drop the use of first tense (so we're consistent with the rest of the README)?